app/nasa_collaboration/page.module.css

```diff
@@ -4,9 +4,35 @@
   height: 100vh;
 }
 
-
 .nasaPicOfTheDayImg {
   width: auto;
   height: 400px;
-
-}
\ No newline at end of file
+}
+.collaborationItem {
```

Comment:
In CSS, the convention is to use [kebab case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) for class names :)